build.gradle

```diff
@@ -16,6 +16,10 @@ archivesBaseName = "fabric-loader"
 def ENV = System.getenv()
 version = version + (ENV.GITHUB_ACTIONS ? "" : "+local")
 
+// For future: Migrate to checkstyle proper.
+// Migration is waiting on some other unmerged code so we can remove the auxiliary gradle file
+apply from: "ideconfig.gradle"
```

Comment:
I think we said to just get rid of this altogeather. Im not sure it even works most of the time when importing the gradle project, and I doubt it fits in with the checkstyle. The editorconfig file should be enough for now.

Reply:
And it is done